cli/src/test_utils.rs

```diff
@@ -46,15 +44,15 @@ pub fn wait_n_slots(rpc_client: &RpcClient, n: u64) -> u64 {
     loop {
         sleep(Duration::from_millis(DEFAULT_MS_PER_SLOT));
         let new_slot = rpc_client.get_slot().unwrap();
-        if new_slot - slot > n {
+        if new_slot.saturating_sub(slot) > n {
             return new_slot;
         }
     }
 }
 
 pub fn wait_for_next_epoch_plus_n_slots(rpc_client: &RpcClient, n: u64) -> (Epoch, u64) {
     let current_epoch = rpc_client.get_epoch_info().unwrap().epoch;
-    let next_epoch = current_epoch + 1;
+    let next_epoch = current_epoch.saturating_add(1);
```

Comment:
This value is only used for the console message in the next statement.
And this function is only used in tests.
So, if this addition would actually saturate, it would only cause the message to be slightly incorrect.
I figured, it probably is not worth complicating this further with an explicit check.